app/assets/javascripts/vote.js

```diff
@@ -0,0 +1,32 @@
+$(function() {
+  var makeAjaxVoteRequest = function(url, data) {
+    var request = $.ajax({
+      method: 'PATCH',
+      data: data,
+      url: url
+    });
+
+    request.done(function(data) {
+      var review = $('#review-' + data.id);
+      review.find(".helpful").text("Helpful: " + data.revup_count)
+      review.find(".not-helpful").text("Not Helpful: " + data.revdown_count)
```

Comment:
Mixed double and single quotes.<br>Identifier 'revdown_count' is not in camel case.<br>Missing semicolon.